packages/ChangesReporting/Annotation/AnnotationExtractor.php

```diff
@@ -28,6 +28,10 @@ public function extractAnnotationFromClass(string $className, string $annotation
         // @see https://regex101.com/r/oYGaWU/1
         $pattern = '#' . preg_quote($annotation, '#') . '\s+(?<content>.*?)$#m';
         $matches = Strings::match($docComment, $pattern);
-        return $matches['content'] ?? null;
+
+        if ($matches && isset($matches['content'])) {
```

Comment:
yeah.. the nette method has not good phpdocs, and therefore its not clear when it returns what.

I gues the `isset` is not required.

```suggestion
        if ($matches) {
```